ui/app/src/components/GlobalAppToolbar/GlobalAppToolbar.tsx

```diff
@@ -76,8 +76,8 @@ export const GlobalAppToolbar = React.memo<GlobalAppToolbarProps>(function(props
       className={clsx(classes.appBar, props.classes?.appBar)}
     >
       <Toolbar className={clsx(classes.toolbar, props.classes?.toolbar)}>
-        <section className={clsx(classes.title, props.classes?.title)}>
-          <Typography variant="h4" component="h1">
+        <section>
+          <Typography variant="h4" component="h1" className={clsx(classes.title, props.classes?.title)}>
```

Comment:
Why? How does this affect the GlobalApp sections that are already using it?

Reply:
It doesn't affect right now (no classes are being applied to current GlobalApp usages), but since this is not going to be used in PublishingStatus I'll revert this change